deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -146,7 +146,16 @@ affinity: {}
 command: []
 
 # Specify additional arguments to pass to the agent binary.
-# Example: `["--strict", "--oneshot"]`
+# For example, to enable JSON logging use `--logging-format`, or
+# to increase the logging verbosity use `--log-level`.
+# The log levels are: 0=Info, 1=Debug, 2=Trace.
+# Use 6-9 for increasingly verbose HTTP request logging.
+# The default log level is 0.
+#
+# Example:
+#  extraArgs:
+#  - --logging-format=json
+#  - --log-level=6 # To enable HTTP request logging
 extraArgs: []
```

Comment:
Matches what is already used in the E2E test:
 * https://github.com/jetstack/jetstack-secure/blob/VC-36032/context-to-client/hack/e2e/values.venafi-kubernetes-agent.yaml#L11-L13

https://github.com/jetstack/jetstack-secure/blob/3b3b68a8b3fd38c3f3b4c70d656c193627da807b/hack/e2e/values.venafi-kubernetes-agent.yaml#L11-L13